tests/platform_tests/api/test_chassis.py

```diff
@@ -128,6 +128,12 @@ def test_get_serial(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost,
         pytest_assert(isinstance(serial, STRING_TYPE), "Chassis serial number appears incorrect")
         self.compare_value_with_device_facts(duthost, 'serial', serial)
 
+    def test_get_serial(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platform_api_conn):
+        duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+        revision = chassis.get_revision(platform_api_conn)
+        pytest_assert(revision is not None, "Unable to retrieve chassis serial number")
```

Comment:
For vendors which does not have it supported do we expect some type of text?
Please note that the same test will run on different branches where this is not supported. so it should not fail and know to distinguish on that.
We have that on other sonic-mgmt test cases. As I don't know the code reference i suggest to look into the code and see maybe Xin Wang comments on different PRs.

Reply:
We expect an empty string for **our** platforms which don't support this information. If this is run on another vendor we could probably expect a NotImplementedError, however that seems to be the case for the other tests in this file which is why there is no additional handling for this function.

Reply:
I am incorrect in my previous comment because the function called here is simply a proxy function that makes a request to the test server which actually makes the call. 

I would presume it safely returns some error which fails this test (this would be expected behavior for a platform which does not implement this method, correct?) given this is how all of the other psu and chassis API tests are performed.